.gitignore

```diff
@@ -26,6 +27,7 @@ local.log
 # ide
 .idea
 *.sublime-*
+.vscode/
```

Comment:
I don't think we want to ignore this folder entirely, right (cc @lobsterkatie)? There are e.g. some debug configs in it that are already in the repo. 
```suggestion
```

Reply:
oh, then lets remove it.